### PR TITLE
tests: improve comments and unix/tcp

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: pkg/server/utils.go [hunk 1]
```diff
@@ -72,34 +66,41 @@ func startSpdkMockupServerOnUnixSocket(rpc *spdkJSONRPC) net.Listener {
 	return ln
 }
 
-func spdkMockServerOnUnixSocket(rpc *spdkJSONRPC, l net.Listener, toSend []string) {
+func spdkMockServerCommunicate(rpc *spdkJSONRPC, l net.Listener, toSend []string) {
 	for _, spdk := range toSend {
+		// wait for client to connect (accept stage)
```
Comment (artek-koltun) [inline pkg/server/utils.go:71]: Usually, a comment in the code it is an indication that a function is missing. Would it make sense to create a couple of small functions?
```
getConnection(....)
readFromClient(...)
substituteIdInResponse(...)
writeToClient(...)
closeWrite(...)
```

File: pkg/server/utils.go [hunk 2]
```diff
@@ -72,34 +66,41 @@ func startSpdkMockupServerOnUnixSocket(rpc *spdkJSONRPC) net.Listener {
...
 		log.Printf("SPDK mockup Server: snd : %s", spdk)
-
+		// send data back to client
 		_, err = fd.Write([]byte(spdk))
 		if err != nil {
 			log.Fatal("Write: ", err)
 		}
-		err = fd.(*net.UnixConn).CloseWrite()
+		// close connection
+		switch fd := fd.(type) {
```
Comment (artek-koltun) [inline pkg/server/utils.go:98]: Suppose `CloseWrite` should be deferred after `fd` creation. I see that we use a `return` above and `log.Fatal` to interrupt the execution... (also if we use `log.Fatal`, all deferred calls are not performed. Would it make sense to use `log.Panic` in this functions instead?)

File: pkg/server/utils.go (hunk 1 above)
Comment (glimchb) [reply pkg/server/utils.go:71]: opened #264

File: pkg/server/utils.go (hunk 2 above)
Comment (glimchb) [reply pkg/server/utils.go:98]: thanks, good catch, fixed with Panic now

**Pull request merged**